core/controllers/feedback.py

```diff
@@ -146,6 +146,17 @@ def post(self, exploration_id):
         self.render_json(self.values)
 
 
+class ReportHandler(base.BaseHandler):
```

Comment:
I thought flagging a exploration will work on the same workflow lines of suggestion/feedback. Currently, only moderators are made aware of the flagging, It can be further extended to send the mails to author of the exploration as well. So, I felt it would be appropriate to put it in feedback.py.
I will change the class name and doctoring.